x-pack/test/api_integration/apis/management/index_management/indices.js

```diff
@@ -0,0 +1,226 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import expect from '@kbn/expect';
+
+import { initElasticsearchHelpers } from './lib';
+import { registerHelpers } from './indices.helpers';
+
+export default function ({ getService }) {
+  const supertest = getService('supertest');
+  const es = getService('es');
+
+  const {
+    createIndex,
+    catIndex,
+    indexStats,
+    cleanUp: cleanUpEsResources
+  } = initElasticsearchHelpers(es);
+
+  const {
+    closeIndex,
+    openIndex,
+    deleteIndex,
+    flushIndex,
+    refreshIndex,
+    forceMerge,
+    freeze,
+    unfreeze,
+    list,
+    reload,
+    clearCache,
+  } = registerHelpers({ supertest });
+
+  describe('indices', () => {
+    after(() => Promise.all([cleanUpEsResources()]));
+
+    describe('clear cache', () => {
+      it('should clear the cache on all indices', async () => {
+        await clearCache('*').expect(200);
+      });
+
+      it('should clear the cache on a single index', async () => {
+        const index = await createIndex();
+        await clearCache(index).expect(200);
+      });
+    });
+
+    describe('close', () => {
+      it('should close an index', async () => {
+        const index = await createIndex();
+
+        // Make sure the index is open
+        const [cat1] = await catIndex(index);
+        expect(cat1.status).to.be('open');
+
+        await closeIndex(index).expect(200);
+
+        // Make sure the index has been closed
+        const [cat2] = await catIndex(index);
+        expect(cat2.status).to.be('close');
+      });
+    });
+
+    describe('open', () => {
+      it('should open an index', async () => {
+        const index = await createIndex();
+
+        await closeIndex(index);
+
+        // Make sure the index is closed
+        const [cat1] = await catIndex(index);
+        expect(cat1.status).to.be('close');
```

Comment:
This should be `closed`

Reply:
That's not what is returned by the endpoint. And the test fails with `closed`

Reply:
My bad, you are correct.  Got confused looking at what is in the UI.